# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformGtk.h
# Commit: b4fee14fe36a
# Full Hash: b4fee14fe36ae084fb392de2a40bcbd9a226bec3
# Author: Martin Stransky <stransky@redhat.com>
# Date: 2020-06-14 09:24:22
# Regressor Bug: 1629788
# File Overlap Count: 1
# Description:
#   Bug 1629788 [Wayland] Add widget.wayland-dmabuf-video-textures.enabled preference, r=jhorak
#   
#   - widget.wayland-dmabuf-video-textures.enabled controls dmabuf texture backend for decoded video frames.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D78293
# ==============================================================================

diff -r 8a463700d169 -r b4fee14fe36a gfx/thebes/gfxPlatformGtk.h
--- a/gfx/thebes/gfxPlatformGtk.h	Sat Jun 13 18:38:38 2020 +0000
+++ b/gfx/thebes/gfxPlatformGtk.h	Sat Jun 13 18:38:45 2020 +0000
@@ -88,6 +88,7 @@
 
 #ifdef MOZ_WAYLAND
   bool UseWaylandDMABufTextures();
+  bool UseWaylandDMABufVideoTextures();
   bool UseWaylandDMABufWebGL();
   bool UseWaylandHardwareVideoDecoding();
 #endif